### ISSUE TITLE
Creating Plugin Doesn't Replace ${UUID}

### ISSUE THREAD

**Issue by philippfromme** [nikku/camunda-modeler-plugins-talk-2019#1]
After executing [step 3](https://github.com/nikku/camunda-modeler-plugins-talk-2019/blob/master/README.md#step-3-generate-your-own-camunda-modeler-plug-in) of the workshop and running `npm run dev` I get an error:

```bash
ERROR in ./client/bpmn-js-extension/index.js 18:17
Module parse failed: Unexpected token (18:17)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| export default {
|   __init__: [ 'pluginService_${UUID}' ],
>   pluginService_${UUID}: [ 'type', ExampleExtensionService ]
| };
|
 @ ./client/index.js 5:0-54 7:21-40
```

`${UUID}` hasn't been replaced when creating the plugin.

**Comment by nikku** [comment]
What's the version of the create plugin you're using? 

`npx create-camunda-modeler-plugin --version`

**Comment by philippfromme** [comment]
`0.0.5` which is the latest.

**Comment by nikku** [comment]
Works for me, must be a Windows issue. I assume other templates in README and package.json are not replaced, either?

**Comment by philippfromme** [comment]
All other templates are replaced. :shit:

**Comment by philippfromme** [comment]
I'm debugging it right now. `replaceAdditional` is never called with `client\bpmn-js-extension\index.js`.

**Comment by nikku** [comment]
Yes. Because of [this line](https://github.com/nikku/create-camunda-modeler-plugin/blob/master/cmd.js#L116). 

We need to normalize the path (replace `\` with `/`) before testing for template replacement. I'll do it.

**Comment by philippfromme** [comment]
I just found it, too. Yes, we do.

**Comment by nikku** [comment]
Should be fixed with `v0.0.6`.

**Comment by nikku** [comment]
Same issue fixed with latest `create-bpmnlint-plugin` generator.
